### PR TITLE
Feature/firebase storage

### PR DESCRIPTION
**Pull request by DekataBG**

### CODE REVIEW

File: DigitalLibrary/DigitalLibrary/MainScreens/BookDetailsScreen/BookDetailsView.swift [hunk 1]
```diff
@@ -12,14 +12,21 @@ struct BookDetailsView: View {
...
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 70, height: 90)
+                    .cornerRadius(8)
+            } else {
+                Image("default_book_cover")
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 70, height: 90)
+                    .cornerRadius(8)
+            }
```
Comment (vasilkadt) [inline DigitalLibrary/DigitalLibrary/MainScreens/BookDetailsScreen/BookDetailsView.swift]: Why are you changing the properties of the image. Here the image should have different from the image at the cell.
```suggestion
                Image(uiImage: image)
                .resizable()
                .scaledToFit()
                .cornerRadius(12)
                .shadow(radius: 5)
                .padding(.horizontal, 80)
                .padding(.vertical, 25)
            } else {
                Image("default_book_cover")
                .resizable()
                .scaledToFit()
                .cornerRadius(12)
                .shadow(radius: 5)
                .padding(.horizontal, 80)
                .padding(.vertical, 25)
            }
```

File: DigitalLibrary/DigitalLibrary/MainScreens/ProfileScreen/ProfileView.swift [hunk 2]
```diff
@@ -9,6 +9,14 @@ import SwiftUI
...
-        Text("Profile")
+        Button("CLick me", action: {
+            let books = BooksMockData.books
+//            ForEach(books) { book in
+//                BooksRepository().create(book)
+//            }
+            books.forEach( { book in
+                BooksRepository().create(book)
+            })
+        })
```
Comment (vasilkadt) [inline DigitalLibrary/DigitalLibrary/MainScreens/ProfileScreen/ProfileView.swift]: What is this?

File: DigitalLibrary/DigitalLibrary/Network/BookCoversRepository.swift [hunk 3]
```diff
@@ -0,0 +1,46 @@
+//
+//  BookCoversRepository.swift
+//  DigitalLibrary
+//
+//  Created by Desislav Adamov on 17.02.24.
+//
+
+import FirebaseStorage
+import Foundation
```
Comment (vasilkadt) [inline DigitalLibrary/DigitalLibrary/Network/BookCoversRepository.swift:9]:
```suggestion

```

File: DigitalLibrary/DigitalLibrary/MainScreens/ProfileScreen/ProfileView.swift (hunk 2 above)
Comment (DekataBG) [reply DigitalLibrary/DigitalLibrary/MainScreens/ProfileScreen/ProfileView.swift]: Forgot to delete it

File: DigitalLibrary/DigitalLibrary/Network/BookCoversRepository.swift (hunk 3 above)
Comment (DekataBG) [reply DigitalLibrary/DigitalLibrary/Network/BookCoversRepository.swift:9]: Foundation is used to for Data

**Review by OgnyanY — APPROVED** [review]

**Pull request merged**